src/main/java/core/basesyntax/service/ServiceWriterImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ServiceWriterImpl implements ServiceWriter {
+    private static final String COMMA = ",";
+    private static final String REPORT_FIRST_LINE = "fruit,quantity";
+
+    @Override
+    public void writeReport(String filePath, Map<Fruit, Integer> storage) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write(REPORT_FIRST_LINE + System.lineSeparator());
+            for (Map.Entry<Fruit, Integer> entry : storage.entrySet()) {
+                writer.write((entry.getKey().getName() + COMMA + entry.getValue()
+                        + System.lineSeparator()));
+            }
```

Comment:
let's create report in `getFruitsReport()` in service. this class is responsible for working with file, particularly writing to file. it should not actually manipulate data

Reply:
what about this?